src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/sentencing/SentencingSynchronisationIntTest.kt

```diff
@@ -132,23 +120,20 @@ class SentencingSynchronisationIntTest : SqsIntegrationTestBase() {
           await untilAsserted {
             mappingApi.verify(getRequestedFor(urlPathEqualTo("/mapping/sentencing/adjustments/nomis-adjustment-category/SENTENCE/nomis-adjustment-id/$NOMIS_ADJUSTMENT_ID")))
           }
-          await untilAsserted { verify(telemetryClient).trackEvent(any(), any(), isNull()) }
```

Comment:
no longer need to wait since next test will not start until current message has finished processing due to the super class surge and wait